src/Api/Monetization/Structure/Reports/Criteria/AbstractCriteria.php

```diff
@@ -125,9 +119,9 @@ public function getApps(): array
     /**
      * @param string ...$appIds
      *
-     * @return \self
+     * @return self
      */
-    public function apps(string ...$appIds): self
+    public function setApps(string ...$appIds): self
```

Comment:
Since the original method names were public APIs, I would suggest  keeping those and may triggering a deprecation error and remove them in the next major version.
I would not just rename (and with that remove) them.

Reply:
@mxr576 
Thank you for the suggestion. 
I agree that keeping the original method names and triggering a deprecation warning is a prudent approach. 
We can then plan to remove the deprecated methods in the next major version. I'll make the necessary changes and update the documentation accordingly.